examples/var/tools/convert_weight.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+
+import numpy as np
+import torch
+
+import mindspore as ms
+
+"""
+Usage: python tools/convert_weight.py \
+            --model_name "var" \
+            --src /path/to/pt/var-d16.pt \
+            --target /path/to/ms/var-d16.ckpt
+
+"""
+
+
+def convert_pt2ms(pt_param, ms_param, pt_weight, save_fp):
+    """#gap PT weights not exists in MS weights, but it's reasonable."""
+    with open(pt_param, "r") as f:
+        f_pt_param = f.readlines()
+    with open(ms_param, "r") as f:
+        f_ms_param = f.readlines()
+    pt_weight = torch.load(pt_weight, map_location=torch.device("cpu"))
+    if "state_dict" in pt_weight:
+        pt_weight = pt_weight["state_dict"]
+
+    ms_weight = []
+
+    assert len(f_pt_param) == len(f_ms_param)
+
```

Comment:
there is no need to do name mapping. can remove the additional 4 txt files.
```
ms_weight = []
for pname in pt_weight:
    ms_name = pname
    ms_val = pt_weight[pt_name].cpu().detach().numpy()
    ms_weight.append({"name": ms_name, "data": ms.Tensor(ms_val)})
```

Reply:
done